string agv_id
---
nist_gear/Model[] faulty_parts
string camera_id
